### PR TITLE
Timestamp test logs

### PR DESCRIPTION
**Pull request by dgquintas**

### CODE REVIEW

**Comment by dgquintas** [comment]
issues #9831 #9399 #9825

File: tools/run_tests/python_utils/jobset.py [hunk 1]
```diff
@@ -128,7 +129,8 @@ def alarm_handler(unused_signum, unused_frame):
     'SKIPPED': 'cyan'
     }
 
-
+FORMAT = '%(asctime)-15s %(message)s'
```
Comment (matt-kwong) [inline tools/run_tests/python_utils/jobset.py]: To go along with the style of constants in this file, FORMAT should be _FORMAT

File: tools/run_tests/python_utils/jobset.py [hunk 2]
```diff
@@ -128,7 +129,8 @@ def alarm_handler(unused_signum, unused_frame):
...
+logging.basicConfig(level=logging.INFO, format=FORMAT)
```
Comment (matt-kwong) [inline tools/run_tests/python_utils/jobset.py]: There should be newlines after this

**Comment by dgquintas** [comment]
Issues: #9797 #9762

**Pull request merged**